### PR TITLE
Migration for Homepage Stats Visualization

### PR DESCRIPTION
**Pull request by srinjoym**
Adds `TickerStat` model with `user_count` and `repo_count` fields

### CODE REVIEW

**Comment by d12** [comment]
I have similar feedback as @BenEmdon , except I'd suggest no default value. A value should always be provided when adding to this table, and a 0 value isn't useful data. Having only a `null: false` validation will ensure there's always a value provided. That's just my opinion, happy to discuss if other have a different opinion :)

File: app/models/ticker_stat.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
+class TickerStat < ApplicationRecord
```
[the remark this answers is not in the document]
Comment (srinjoym) [reply app/models/ticker_stat.rb]: Whoops, good catch, removed the model!

**Comment by BenEmdon** [comment]
@srinjoym are we still taking this approach?

**Comment by srinjoym** [comment]
Closing out because we decided to just use the latest user and repo record counts on the homepage

**Pull request closed without merging**